test/missing.jl

```diff
@@ -527,3 +527,10 @@ mutable struct Obj; x; end
     @test ismissing(wref[1] == missing)
     @test ismissing(missing == wref[1])
 end
+
+@testset "hasmissing" begin
+    @test hasmissing([1, 2, 3, 4]) == false
+    @test hasmissing([1, 2, 3, missing, 5]) == true
+    @test hasmissing(skipmissing(skipmissing([1, 2, 3, 4]))) == false
```

Comment:
Why two `skipmissings`?

Reply:
I wanted to test if it worked on nested iterators. But I'll remove that.